libs/platform/src/lib/components/list/list.component.html

```diff
@@ -5,7 +5,8 @@
     (keydown)="_keyManager.onKeydown($event);_handleKeyDown($event);" (change)="_onSelectionChanged($event)"
     [attr.aria-setsize]="ariaSetsize" [attr.aria-label]="ariaLabel" [attr.aria-multiselectable]="ariaMultiselectable">
     <fd-busy-indicator *ngIf="loadMore && loadOnScroll" size="l"
-        class="fd-busy-indicator-custom fd-list__title--growing" [block]="true" [loading]="_loading" role="alert" [attr.aria-busy]="_loading" aria-label="loading">
+        class="fd-busy-indicator-custom fd-list__title--growing" [block]="true" [loading]="_loading" role="alert"
+   [attr.aria-busy]="_loading" i18n-aria-labelledby="@@platform.list.loadOnScroll" [attr.aria-labelledby]='_loading ? "loading" : ""'>
```

Comment:
you can not tranlsate `aria-labelledby` attribute cause it should refer to some element id, please drop `18n-aria-labelledby`